Source/Core/RequestScheduler.js

```diff
@@ -32,6 +55,70 @@ define([
     function RequestScheduler() {
     }
 
+    function handleLeftoverRequest(url, type, distance) {
+        // TODO : don't create a new object, reuse instead
+        leftoverRequests.push({
+            url : url,
+            type : type,
+            distance : defaultValue(distance, 0.0)
+        });
+    }
+
+    function printStats() {
+        if (stats.numberOfRequestsThisFrame > 0) {
+            console.log('Number of requests attempted: ' + stats.numberOfRequestsThisFrame);
+        }
+
+        var numberOfActiveRequests = RequestScheduler.maximumRequests - RequestScheduler.getNumberOfAvailableRequests();
+        if (numberOfActiveRequests > 0) {
+            console.log('Number of active requests: ' + numberOfActiveRequests);
+        }
+    }
+
+    function distanceSortFunction(a, b) {
+        return a.distance - b.distance;
+    }
+
+    RequestScheduler.resetBudgets = function() {
+        // TODO : Right now, assume that requests made the previous frame will probably be issued again the current frame
+        // TODO : If this assumption changes, we should introduce stealing from other budgets.
+
+        if (!RequestScheduler.prioritize) {
+            return;
+        }
+
+        printStats();
```

Comment:
Remove this.  Instead, for example, the Sandcastle example could call this each frame (or it could be commented out in the Sandcastle example).